plugins/catalog/src/components/CatalogTable/CatalogTable.tsx

```diff
@@ -116,8 +123,8 @@ export const CatalogTable = (props: CatalogTableProps) => {
       const isStarred = isStarredEntity(entity);
       return {
         cellStyle: { paddingLeft: '1em' },
-        icon: () => favoriteEntityIcon(isStarred),
-        tooltip: favoriteEntityTooltip(isStarred),
+        icon: () => (isStarred ? <YellowStar /> : <StarBorder />),
```

Comment:
i'd suggest `icon: () => <FavoriteEntity starred={isStarred} />` and introducing that as an optional force-set-to-true-or-false prop. It even has the tooltip, so you can remove that from the next line

same below obvs

Reply:
Btw, I think the proper way forward for these might be to add them as system icons, later x)

Reply:
Yeah could do something like that

Reply:
Let's merge as is and figure out the way forward in an issue

Reply:
#9948